testing/jormungandr-integration-tests/src/common/jcli/services/certificate_builder.rs

```diff
@@ -14,6 +14,7 @@ impl CertificateBuilder {
     pub fn new(jcli: JCli) -> Self {
         Self { jcli }
     }
+    #[allow(clippy::too_many_arguments)]
```

Comment:
Maybe clippy is right here and there should be a proper builder API to something that's already called `CertificateBuilder`. Cc @dkijania

Reply:
Spun off into #2703, let's go with the lint override for now.

Reply:
Spun off into #2703, let's go with the lint override for now.